test/unit/gateways/kushki_test.rb

```diff
@@ -280,6 +280,26 @@ def test_failed_refund
     assert_equal 'K010', refund.error_code
   end
 
+  def test_partial_refund
```

Comment:
Should we also add a test that shows that we are not sending any body for a full refund?

Reply:
ok will do